tests/Microsoft.Identity.Test.Integration.net45/SeleniumTests/InteractiveFlowTests.cs

```diff
@@ -211,9 +229,12 @@ private async Task<AuthenticationResult> RunTestForUserAsync(LabResponse labResp
             userCacheAccess.AssertAccessCounts(3, 3);
             Assert.IsFalse(userCacheAccess.LastNotificationArgs.IsApplicationCache);
 
+            AssertTelemetryHeaders(factory, LastRequestExpectedTelemetryApiCorId);

```

Comment:
It would be easier if we had a higher level assertion call similar to `userCacheAccess.AssertAccessCounts(3, 3)`. See prev comment